config/a8c-for-agencies-development.json

```diff
@@ -0,0 +1,26 @@
+{
+	"env": "development",
+	"env_id": "a8c-for-agencies-development",
+	"favicon_url": "/calypso/images/favicons/favicon-jetpack-cloud-development.ico",
+	"client_slug": "browser",
+	"protocol": "http",
+	"hostname": "automattic.agencies.localhost",
+	"i18n_default_locale_slug": "en",
+	"port": 3000,
+	"site_name": "Automattic For Agencies",
+	"features": {
+		"a8c-for-agencies": true,
+		"always_use_logout_url": true,
+		"dev/auth-helper": true,
+		"dev/preferences-helper": true
+	},
+	"enable_all_sections": false,
+	"sections": {
+		"a8c-for-agencies": true
+	},
+	"site_filter": [ "jetpack" ],
+	"theme": "jetpack-cloud",
```

Comment:
I'm not sure the significance of this right now but should this be updated to a8c-for-agencies as well to match the stage and production config

Reply:
Thanks for the review, @oikeme! Since we don't have a theme for `a8c-for-agencies`, I have updated it all to `jetpack-cloud`: https://github.com/Automattic/wp-calypso/pull/87292/commits/a9919d89324115980a2c2d14b826625b21498589